ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -102,6 +102,49 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlPostWithSession(SteamCommunityURL, request, data, referer).ConfigureAwait(false);
 		}
 
+		internal async Task AcceptDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				Bot.ArchiLogger.LogNullError(nameof(response));
+				return;
+			}
+
+			const string requestRedeemGift = "/gifts/0/resolvegiftcard";
+			HtmlNodeCollection nodes = response.DocumentNode.SelectNodes("//div[starts-with(@id,'pending_gift_')]");
```

Comment:
This is going to be expensive, how the sample HTML looks like? Isn't there a better way to narrow this down?

Reply:
![image](https://user-images.githubusercontent.com/24438339/44313926-b6a1e100-a419-11e8-8a02-ecd398a46b17.png)
So I guess we can retrieve GID only this way

Reply:
Yes, but in this case `"//div[@class='pending_gift']/div[starts-with(@id,'pending_gift_')]"` is at least ten times better as you're doing `Equals()` checks and not much more expensive `StartsWith()`.